opendata_collector_module/metrics_opendata_collector/logger_manager.py

```diff
@@ -0,0 +1,127 @@
+#  The MIT License
+#  Copyright (c) 2021- Nordic Institute for Interoperability Solutions (NIIS)
+#  Copyright (c) 2017-2020 Estonian Information System Authority (RIA)
+#
+#  Permission is hereby granted, free of charge, to any person obtaining a copy
+#  of this software and associated documentation files (the "Software"), to deal
+#  in the Software without restriction, including without limitation the rights
+#  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+#  copies of the Software, and to permit persons to whom the Software is
+#  furnished to do so, subject to the following conditions:
+#
+#  The above copyright notice and this permission notice shall be included in
+#  all copies or substantial portions of the Software.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+#  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+#  THE SOFTWARE.
+
+import json
+import logging
+import os
+import time
+from logging.handlers import WatchedFileHandler
+
+
+def get_timestamp():
+    return int(time.time())
+
+
+def get_local_timestamp():
+    return time.strftime('%Y-%m-%d %H:%M:%S %z', time.localtime())
```

Comment:
Type hints would be nice. Also why not keeping it as a constant as well?

Reply:
Added